pydrive2/fs/spec.py

```diff
@@ -542,6 +542,35 @@ def cp_file(self, lpath, rpath, **kwargs):
             buffer = io.BytesIO(stream.read())
             self.upload_fobj(buffer, rpath)
 
+    @_gdrive_retry
+    def mv(self, path1, path2, recursive=False, maxdepth=None, **kwargs):
+
+        if recursive:
+            raise NotImplementedError("Recursive move is not supported")
+        if maxdepth:
+            raise NotImplementedError("Max depth move is not supported")
+
+        file1_name = posixpath.basename(path1)
+        file2_name = posixpath.basename(path2)
+
+        file1_parent = self._parent(path1)
+        file2_parent = self._parent(path2)
```

Comment:
Looked at the code - so it can be empty I think (if `path2` is `root` for example). Need to check that everything works in that case.

Reply:
it does work if you specify the name of the target:

```python
fs.mv('root/tmp/a.pdf', 'root/b.pdf')
```

but you do:

```python
fs.mv('root/tmp/a.pdf', 'root/)
```

it will throw an error.

Reply:
it should be working though, right? It should be close to `mv` UNIX semantics.

Reply:
Fixed.
Now it works with or without the file name, and with or without the leading slash:

```python
fs.mv('root/tmp/b.pdf', 'root/')
fs.mv('root/tmp/b.pdf', 'root')
fs.mv('root/tmp/b.pdf', 'root/a.pdf')
```